astropy_helpers

```diff
@@ -1 +1 @@
-Subproject commit fedd7c52ba80e2246f83b8d1752baa3a153a33e2
+Subproject commit 160b48587680e11f12b66a51641a2601b869d1fa
```

Comment:
Should not be changed.  This commit is occurring in many of your PRs.  Please consider following the advice posted in reply to this StackOverflow question

http://stackoverflow.com/questions/18276951/how-do-i-stop-git-from-tracking-any-changes-to-a-file-from-this-commit-forward